### DISCUSSION TITLE
Feature Request: Payment expiration for a fixed date

### DISCUSSION THREAD

**Discussion by digitalyours** [pretix/pretix#2026] [Ideas]
Hey Pretix-Team,

We're currently using Pretix for an event registration with bank transfer only. For our scenario (youth events) it would be beneficial to define a "fixed payment date" rather than "payment days". Usually we have quite a long lead time between registration and when the payment is finally due. 

Best
@bockstaller & Christoph

**Comment by raphaelm** [comment]
You can just set "payment term in days" to "999999" and configure a last date of payment. I think we don't need to add an additional complication to the settings UI for this.
